### PR TITLE
Add non-nullable contained_level_page column

### PR DESCRIPTION
**Pull request by maddiedierker**
Follow-up to #29443. Fixes [this Honeybadger error](https://app.honeybadger.io/projects/45435/faults/51801323).

The `contained_level_page` column on the `contained_levels` table is non-nullable, but was null for BubbleChoice levels, so inserting new rows was failing. This fixes that failure by hardcoding the `contained_level_page` value to 1 for BubbleChoice levels.

### CODE REVIEW

**Comment by bencodeorg** [comment]
👍 Cool if I run this manually once you've merged to unblock Baker on some work he was hoping to do before the end of the week?

**Comment by bencodeorg** [comment]
Whoa sorry, didn't realize "Close and comment" meant "Close this PR"!

**Review by bencodeorg — APPROVED** [review]

**Comment by maddiedierker** [comment]
@bencodeorg absolutely! lol it's okay, the "close and comment" button is... not very intuitive

**Comment by bencodeorg** [comment]
Ran manually, looks like it worked 👌

**Pull request merged**
